packages/vite/src/node/server/moduleGraph.ts

```diff
@@ -131,18 +153,41 @@ export class ModuleGraph {
     timestamp: number = Date.now(),
     isHmr: boolean = false,
     hmrBoundaries: ModuleNode[] = [],
+    softInvalidated = false,
```

Comment:
maybe a name more "action that state" like `softInvalidate`

Reply:
Ah yeah it's an artifact from some refactors, will update